frontend/src/lit-components/pages/nav.js

```diff
@@ -0,0 +1,40 @@
+import { html, LitElement } from 'lit';
+
+import { NavStyles } from '../../styles';
+
+export class NavBar extends LitElement {
+    static styles = NavStyles;
+
+    static properties = {
+
+    }
+
+    constructor() {
+        super();
+    }
+
+    render() {
+        return html`
+        <header>
+            <a href="#" id="profile">
+                <svg width="60" height="60" viewBox="0 0 41 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Had trouble using the img tag. This was the simplest fix I found. If anyone knows a better way please shout

Reply:
I'm guessing the img tag was not working because our express server hasn't been configured to serve the images from public/images yet.
I pushed a fix for it to this PR: https://github.com/aaronsh1/food-recipe-suggestions/pull/5

Reply:
I imported the image into my component.  You can have a look at the 'feature/profile' branch in the profile-page component

Reply:
We're using Rollup to bundle our application as a single js file. Images which get imported directly are added to the bundle as base64 encoded images. This increases the size of the bundle and can greatly increase page load times.

I have updated the frontend express server to be able to serve images, so you can simply point your src attribute to `/public/images/my_image.jpg`.
The benefit of having express serve the images over the images being imported is that the images then only get downloaded once they are needed, instead of being downloaded with the bundled script.